apps/condo/domains/banking/access/BankIntegrationAccessRight.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Generated by `createschema banking.BankIntegrationAccessRight 'integration:Relationship:BankIntegration:PROTECT; user:Relationship:User:CASCADE;'`
+ */
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+
+async function canReadBankIntegrationAccessRights (args) {
+    const { authentication: { item: user } } = args
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return {}
+
+    return { user: { id: user.id }, deletedAt: null }
```

Comment:
```suggestion
    return { user: { id: user.id, deletedAt: null }, deletedAt: null }
```

Reply:
does not make sense, because if the user is deleted, the guard will work at the beginning and the request will be rejected without reaching this check